app/controllers/posts_controller.rb

```diff
@@ -1,6 +1,7 @@
 class PostsController < ApplicationController
   def index
     @user = User.find(params[:user_id])
+    # @posts = @user.posts.page(params[:page].per(3))
```

Comment:
- [ ] Comments in our within our codes renders codebase messy. It will be nice if we can remove them before pushing to the repo :+1: Please also check comments controller file.